packages/forms-web-app/src/dynamic-forms/dynamic-components/list-add-more/question.js

```diff
@@ -319,17 +320,29 @@ class ListAddMoreQuestion extends Question {
 
 	/**
 	 * removes answer with answerId from response if present
+	 * @param {import('express').Request} req
 	 * @param {JourneyResponse} journeyResponse
 	 * @param {string} answerId
-	 * @returns {JourneyResponse} updated JourneyResponse
+	 * @returns {Promise<JourneyResponse | boolean> } updated JourneyResponse
 	 */
-	async removeAction(journeyResponse, answerId) {
+	async removeAction(req, journeyResponse, answerId) {
 		let responseToSave = {
 			answers: {
 				[this.fieldName]: []
 			}
 		};
 
+		const isAddressQuestion = this.subQuestion instanceof AddressAddMoreQuestion;
+		if (isAddressQuestion) {
+			const updatedLPA = await req.appealsApiClient.deleteSubmissionAddress(
+				journeyResponse.referenceId,
+				answerId
+			);
+			console.log(journeyResponse);
```

Comment:
oops?

Reply:
Let's put this one down as a cartographer's deliberate error